Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -92,14 +142,42 @@ pathManagerState& cruisingState::getInstance()
 
 void coordinateTurnElevation::execute(pathManager* pathMgr)
 {
-    //get elevation and turning data
+    
+    SFOutput_t * pSensFusionOutput = sensorFusion::GetSFOutput(); // Get sensor fusion data
+    _WaypointManager_Data_Out * waypointOutput = cruisingState::GetOutputData(); // Get output data from waypoint manager
```

Comment:
how are we gonna implement commands from other stages? We gotta agree on some sort of system.